lib/hammer_cli_csv/host_groups.rb

```diff
@@ -45,66 +47,77 @@ def export
             realm = hostgroup['realm_name']
             puppet_proxy = hostgroup['puppet_proxy_id'] ? foreman_host(:id => hostgroup['puppet_proxy_id']) : nil
             puppet_ca_proxy = hostgroup['puppet_ca_proxy_id'] ? foreman_host(:id => hostgroup['puppet_ca_proxy_id']) : nil
-            # TODO: http://projects.theforeman.org/issues/7597
-            # content_source = hostgroup['content_source_id'] ? foreman_host(:id => hostgroup['content_source_id']) : nil
-            content_source = nil
+            content_source = hostgroup['content_source_id'] ? foreman_host(:id => hostgroup['content_source_id']) : nil
```

Comment:
the TODO was removed, but it looks like the ticket itself is still in NEW state. Is the TODO no longer relevant?